docker/local/turbinia-local-template.conf

```diff
@@ -0,0 +1,207 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is there a good way we could use the existing config for this?  I'm not sure how divergent this is from the main config, but this is now the third place that we have to remember to update when the main config file gets updated, so it would be nice if we didn't have the duplication if there's a good way around it.

Reply:
I could take the default configuration and sed the correct values in the template so a local stack can boot? The reason I added a template is because I don't want people to have to take the generic template and adjust all the values so they can get it to work.

Reply:
Yeah, using some kind of template (generated or static) is a good idea.  Using sed might be the best quick solution, although it is a little hacky too.  We could implement this as actual templates, but that's probably overkill.   I just know that there are going to be cases where we update one of the config templates and not the other (because it's happened at least twice already with the version in the terraform config), and wanted to try to avoid that if possible.

Reply:
yeah, agree, having 1 template and using that is better. Added a sed expression file and adjusted the documentation accordingly.